components/typography/styles.js

```diff
@@ -2,19 +2,45 @@ import '../colors/colors.js';
 import { css, unsafeCSS } from 'lit';
 
 export const _isValidCssSelector = (selector) => {
-	if (selector === ':host') return true;
-	const re = /([a-zA-Z0-9-_ >.#]+)(\[[a-zA-Z0-9-_]+\])?([a-zA-Z0-9-_ >.#]+)?/g;
-	const match = selector.match(re);
+	const partIsValid = (part) => {
+		const re = /([a-zA-Z0-9-_ >.#]+)(\[[a-zA-Z0-9-_]+\])?([a-zA-Z0-9-_ >.#]+)?/g;
+		if (part === ':host') return true;
+		const match = part.match(re);
+		const isValid = !!match && match.length === 1 && match[0].length === part.length;
+		if (!isValid) {
+			console.warn(`Invalid CSS selector: "${part}"`);
+		}
+		return isValid;
+	};
 
-	return !!match && match.length === 1 && match[0].length === selector.length;
+	const parts = selector.split(',');
+	const allValid = parts.every(part => partIsValid(part));
+	return allValid;
```

Comment:
This function still doesn't cover everything, but this update just allows sets of selectors, i.e. `.class1, .class2` which BSI needs.